tests/common.sh

```diff
@@ -2,46 +2,131 @@
 
 get_nolines()
 {
-  zcat $1 2> /dev/null | wc -l
+    zcat $1 2> /dev/null | wc -l
 }
 
-download_warc()
+download_file()
 {
-    warc=$1
-    remote=$2
-    if [ ! -f "${warc}" ]; then
-        wget -q "${remote}" -O "${warc}"
+    local path=$1
+    local remote=$2
+
+    if [ ! -f "${path}" ]; then
+        wget -q "${remote}" -O "${path}"
     fi
 }
 
 download_monocleaner_model()
```

Comment:
I didn't know, and it's a good idea (thank you for the tip!), but I think that for this case it'd be better to keep this way in order to avoid the tests to fail if, for some reason, monocleaner is not installed. I know it will fail anyway, but the specific tests which use monocleaner, not all of them.